src/pfUI.cpp

```diff
@@ -114,19 +114,19 @@ void BlinkCoord(short ax, short ay, bool signDir) {
 	setColor(lightGrey, dbc);
 	gotoXY((scrW - tmp2.length()) / 2, 9);
 	cout << tmp2;
-	Sleep(rand() % 250);
+	Sleep(rng() % 250);
```

Comment:
`rng` is not declared here and will cause compile error.
Please at lease compile before you commit!

Reply:
fixed qwq